--- sysstat-10.1.2/rndr_stats.c.orig	2012-10-06 08:21:47.083578966 -0400
+++ sysstat-10.1.2/rndr_stats.c	2012-10-06 08:21:52.113445453 -0400
@@ -125,10 +125,10 @@
 			/* Got format args? */
 			switch(mid->t) {
 			case iv:
-				printf(txt[isdb], mid->a.i, mid->b.i);
+				printf("%s", txt[isdb], mid->a.i, mid->b.i);
 				break;
 			case sv:
-				printf(txt[isdb], mid->a.s, mid->b.s);
+				printf("%s", txt[isdb], mid->a.s, mid->b.s);
 				break;
 			}
 		}
@@ -137,7 +137,7 @@
 			 * Additional NULL parameter below works around
 			 * fatal error when compiled with -Werror=format-security.
 			 */
-			printf(txt[isdb], NULL);	/* No args */
+			printf("%s", txt[isdb], NULL);	/* No args */
 		}
 	}
 
